tests/nightly/model_backwards_compatibility_check/lm_rnn_gluon_inference.py

```diff
@@ -17,197 +17,31 @@
-                raise ValueError("Invalid mode %s. Options are rnn_relu, "
-                                 "rnn_tanh, lstm, and gru"%mode)
-            if tie_weights:
-                self.decoder = nn.Dense(vocab_size, in_units = num_hidden,
-                                        params = self.encoder.params)
-            else:
-                self.decoder = nn.Dense(vocab_size, in_units = num_hidden)
-            self.num_hidden = num_hidden
-
-    def forward(self, inputs, hidden):
-        emb = self.drop(self.encoder(inputs))
-        output, hidden = self.rnn(emb, hidden)
-        output = self.drop(output)
-        decoded = self.decoder(output.reshape((-1, self.num_hidden)))
-        return decoded, hidden
-
-    def begin_state(self, *args, **kwargs):
-        return self.rnn.begin_state(*args, **kwargs)
 
 context = mx.cpu(0)
 
-def batchify(data, batch_size):
-    """Reshape data into (num_example, batch_size)"""
-    nbatch = data.shape[0] // batch_size
-    data = data[:nbatch * batch_size]
-    data = data.reshape((batch_size, nbatch)).T
-    return data
-
-def get_batch(source, i):
-    seq_len = min(args_bptt, source.shape[0] - 1 - i)
-    data = source[i : i + seq_len]
-    target = source[i + 1 : i + 1 + seq_len]
-    return data, target.reshape((-1,))
-
-def detach(hidden):
-    if isinstance(hidden, (tuple, list)):
-        hidden = [i.detach() for i in hidden]
-    else:
-        hidden = hidden.detach()
-    return hidden
-
-def eval(data_source, model):
-    total_L = 0.0
-    ntotal = 0
-    loss = gluon.loss.SoftmaxCrossEntropyLoss()
-    hidden = model.begin_state(func = mx.nd.zeros, batch_size = args_batch_size, ctx=context)
-    for i in range(0, data_source.shape[0] - 1, args_bptt):
-        data, target = get_batch(data_source, i)
-        output, hidden = model(data, hidden)
-        L = loss(output, target)
-        total_L += mx.nd.sum(L).asscalar()
-        ntotal += L.size
-    return total_L / ntotal
-
 def test(test_data, model):
-    test_L = eval(test_data, model)
-    return test_L, math.exp(test_L)
+	test_L = eval(test_data, model)
+	return test_L, np.exp(test_L)
 
 def get_top_level_folders_in_bucket(s3client, bucket_name):
-    '''This function returns the top level folders in the S3Bucket. These folders help us to navigate to the trained model files stored for different MXNet versions. '''
-    bucket = s3client.Bucket(bucket_name)
-    result = bucket.meta.client.list_objects(Bucket=bucket.name,
-                                         Delimiter=backslash)
-    folder_list = list()
-    for obj in result['CommonPrefixes']:
-        folder_list.append(obj['Prefix'].strip(backslash))
+	'''This function returns the top level folders in the S3Bucket. These folders help us to navigate to the trained model files stored for different MXNet versions. '''
+	bucket = s3client.Bucket(bucket_name)
+	result = bucket.meta.client.list_objects(Bucket=bucket.name,
+										 Delimiter=backslash)
```

Comment:
fix indent

Reply:
Done. Fixed in : 34853525c99a72c114c2f754403de421b0854409